bcipy/helpers/stimuli.py

```diff
@@ -480,21 +481,30 @@ def best_case_rsvp_inq_gen(alp: list,
     return InquirySchedule(samples, times, colors)
 
 
-def calibration_inquiry_generator(
+NO_TARGET_INDEX = None
+
+
+def generate_calibration_inquiries(
         alp: List[str],
-        timing: List[float] = [0.5, 1, 0.2],
+        timing: List[float] = None,
         jitter: Optional[int] = None,
-        color: List[str] = ['green', 'red', 'white'],
-        stim_number: int = 10,
-        stim_length: int = 10,
+        color: List[str] = None,
+        inquiry_count: int = 100,
+        stim_per_inquiry: int = 10,
         stim_order: StimuliOrder = StimuliOrder.RANDOM,
         target_positions: TargetPositions = TargetPositions.RANDOM,
-        nontarget_inquiries: int = 10,
+        percentage_without_target: int = 10,
```

Comment:
Based on our previous discussions maybe we can scrap this number and set the default as 0?